### ISSUE TITLE
LightCurveFiles from tpf.interact lack SAP_FLUX label

### ISSUE THREAD

**Issue by christinahedges** [KeplerGO/lightkurve#556]
We currently can't use the `LightCurveFiles` created by `tpf.interact` because the flux extension isn't readable. 

We can fix this by either changing the `LightCurveFile` class to be able to read a "FLUX" extension from the fits file, or changing `tpf.interact` to output fits files with "SAP_FLUX" as an extension. I think I prefer the second option. 

![image](https://user-images.githubusercontent.com/14965634/62320323-c19c2880-b454-11e9-9b5d-01d3ec4b23f6.png)

**Comment by barentsen** [comment]
I think it makes sense for `interact` to write a `SAP_FLUX` extension, because it is using simple aperture photometry after all.  @gully are you happy with that?  Do you want to go ahead and make that change?

**Comment by gully** [comment]
I had some reservation about overloading `SAP_FLUX`, which has historically mapped onto the pipeline aperture.  I also had some reservation, since some degree of pre-processing could have been performed.

Since some methods expect `SAP_FLUX`, though, it indeed may be useful to just define it for convenience.  What methods presently expect SAP_FLUX?

**Comment by christinahedges** [comment]
It's mostly just so we can interact with it like a pipeline file, i.e. `lcf.SAP_FLUX` is something people might expect to do, because it's in our docs. Alternately we could allow for `lcf.FLUX`?

**Comment by barentsen** [comment]
My two cents is that the *Lightkurve philosophy* is to make working with the data as easy as possible, while trusting the user to be smart enough to understand where data comes from and what the algorithms are doing.

I think the implication is to adopt `SAP_FLUX` as the standard extension name for simple aperture photometry, and leave it to the user to understand where the file they are using came from.

**Comment by gully** [comment]
OK, `FLUX` is currently hardcoded into `to_fits()`:

```python
        def _make_lightcurve_extension(extra_data={}):
            """Create the 'LIGHTCURVE' extension (i.e. extension #1)."""
            # Turn the data arrays into fits columns and initialize the HDU
            cols = []
            if ~np.asarray(['TIME' in k.upper() for k in extra_data.keys()]).any():
                cols.append(fits.Column(name='TIME', format='D', unit=self.time_format,
                                        array=self.time))
            if ~np.asarray(['FLUX' in k.upper() for k in extra_data.keys()]).any():
                cols.append(fits.Column(name='FLUX', format='E',
                                        unit='counts', array=self.flux))
```

So to make this change, I will:
1. Make a kawrg for to_fits with `FLUX` as the default.
2. Make interact pass in `SAP_FLUX`.
3. Spot check unit tests.

**Comment by gully** [comment]
Closed by #559
